firestore-bigquery-export/guides/IMPORT_EXISTING_DOCUMENTS.md

```diff
@@ -1,4 +1,4 @@
-The `fs-bq-import-collection` script is for use with the official Firebase Extension [**Export Collections to BiqQuery**](https://github.com/firebase/extensions/tree/master/firestore-bigquery-export).
+The `fs-bq-import-collection` script is for use with the official Firebase Extension [**Export Collections to BiqQuery**](https://github.com/firebase/extensions/tree/master/firestore-bigquery-export). You can use this script instead of enabling the extension's built-in backfill feature.
```

Comment:
Maybe say something like "We recommend using the automatic import instead, but this script can be used instead" or something like that. (Do we still need the script?)

Reply:
Got this wording from Kevin. I think we should first deprecate the script, then remove it at a later date. Does that seem reasonable?

Reply:
Currently the import script supports collection group queries for imports, is this something we need to consider as part of the backfill?

Reply:
We definitely should release the backfill at parity with the import script. I'll try to add support for it, thought I'd appreciate your help testing it since I don't have much context on how they work

Reply:
Added.